Composer/packages/server/src/controllers/orchestrator.ts

```diff
@@ -36,28 +39,38 @@ async function getModelList(): Promise<IOrchestratorNLRList> {
 
 let state: DownloadState = DownloadState.STOPPED;
 
-interface DefaultModelRequest {
-  language: 'en' | 'multilang';
-}
-
-function isDefaultModelRequest(arg: any): arg is DefaultModelRequest {
-  return arg !== undefined;
+function isValidModelRequest(arg: any): arg is IOrchestratorModelRequest {
```

Comment:
use the type instead of any

Reply:
This typeguard is there since arg could indeed be anything coming from client.